### PR TITLE
add chain info hash in a digest header

### PR DESCRIPTION
**Pull request by willscott**

### CODE REVIEW

File: http/server.go [hunk 1]
```diff
@@ -300,6 +301,7 @@ func (h *handler) ChainInfo(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Cache-Control", "public, max-age=604800, immutable")
 	w.Header().Set("Expires", time.Now().Add(7*24*time.Hour).Format(http.TimeFormat))
 	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Digest", "drand="+hex.EncodeToString(info.Hash()))
```
Comment (Kubuxu) [inline http/server.go:304]: This should probably be `X-Digest` as it isn't standard header.

Comment (willscott) [reply http/server.go:304]: it is a standard header, just not a standard algorithm 😁  Maybe "x-drand"?

Comment (willscott) [reply http/server.go:304]: ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Digest

Comment (Kubuxu) [reply http/server.go:304]: Hmm, right, maybe `x-info-hash`

**Comment by willscott** [comment]
closing in favor of #435

**Pull request closed without merging**